collections/Canon_Pat_Lat/MS_Canon_Pat_Lat_188.xml

```diff
@@ -42,23 +41,81 @@
+                        <locus>(fols. 75v–121v)</locus>
+                        <note>I Corinthians</note>
+                        <msItem>
+                           <note>Argument</note>
+                           <rubric>Incipit argumentum epistolę primę ad Corintheos</rubric>
+                           <incipit>Precepto domini salvatoris admonitus apostolus Paulus venit Corinthum</incipit>
+                           <explicit>&amp; reliquos confirmare.</explicit>
+                        </msItem>
+                        <rubric>Incipit epistola primam ad Corintheos</rubric>
+                        <incipit>Paulus apostolus vocatus Ihesu Christi per voluntatem dei subauditur patris. De nomine Pauli</incipit>
+                        <explicit>vel vera sint omnia superius a me memorata.</explicit>
+                        <finalRubric>Explicit epistola prima.</finalRubric>
+                        <bibl><title>PL</title>, cxvii, 508–606; Stegmüller, <title>Rep. Bibl.</title>, no. 3102</bibl>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 121v–149v)</locus>
+                        <note>II Corinthians</note>
+                        <msItem>
+                           <note>Argument</note>
+                           <rubric>Ad Corintheos incipit argumentum IIª epistolę</rubric>
+                           <incipit>Apostolo recedente a Corinthiis cum audisset eos seductos a pseudo apostolis</incipit>
+                           <explicit>hortaturque proficere semper eos ad meliora.</explicit>
+                        </msItem>
+                        <rubric>Incipit epistola secunda ad Corintheos</rubric>
+                        <incipit>Paulus apostolus Ihesu Christi. Ex hoc satis iam in superioribus dictum est</incipit>
+                        <explicit>&amp; communicatio id est societas &amp; coniunctio spiritus sancti cum omnibus vobis sub(audis) sit. Amen, fiat ut diximus.</explicit>
+                        <note>In the incipit ‘Christi’ is a variant of the usual XPC Christogram: ‘XPYCTI’.</note>
+                        <bibl><title>PL</title>, cxvii, 605–68; Stegmüller, <title>Rep. Bibl.</title>, no. 3103.</bibl>
+                     </msItem>
+                     <note>Rubrics throughout include indications of feast days for the Temporale, Sanctorale, and Common, e.g. ‘Dom. VIII post Pentecosten’ (fol. 34r), ‘Dom. IIII post Pentecosten’ (fol. 35r), ‘In nat. S. Andree apostoli’ (fol. 44v), ‘In dedicatione ęcclesię’ (fol. 82v), ‘De martyribus’ (fol. 87r).</note>
                   </msItem>
                </msContents>
                <physDesc>
                   <objectDesc form="codex">
                      <supportDesc material="perg">
                         <support>parchment</support>
+                        <extent>i (modern paper) + 149 + 1 + i (modern paper)</extent>
+                        <foliation>[i], 1–[150]</foliation>
```

Comment:
if the foliation has been done the square brackets can be removed; it will be helpful to say something like "fols. i, 150 foliated June 2021", since the foliatoin will not be present in the digital images